### PR TITLE
instructor>students tab : Mention 'click on the panels to expand' #2773

### LINKED ISSUE #2773: instructor>students tab : Mention 'click on the panels to expand'
Add the following line just above the first panel. 
'Click on the panels below to expand'

### PR DESCRIPTION
**Pull request by JosephineKwa**
Fixes #2773

### CODE REVIEW

**Comment by JosephineKwa** [comment]
Sorry @kanghj ! I accidentally closed the earlier pull request by accident!

**Comment by damithc** [comment]
@JosephineKwa I think you can reopen closed PRs. Anyway, can you also post a screenshot of the updated UI?

**Comment by JosephineKwa** [comment]
Here is the change that I made. Above the panels I added the instructions to click on them to expand. 
screenshot 30

**Comment by damithc** [comment]
Can we have more space above the text than below it (so that it appears closer to panels below than the panel above)?
I wonder if we should use gray color text instead, to make it less conspicuous. 
What happens when there are no students under this instructor?

**Comment by JosephineKwa** [comment]
Ok, I'll try it out and post the results here. 

If there are no students under this instructor, the panel still remains, thus the text will still stay there.
The panel expands but no students are displayed.

screenshot 32

**Comment by damithc** [comment]
Sorry, I meant 'what if there are no courses under the instructor?'.

**Comment by JosephineKwa** [comment]
Yup! If there aren't any courses the instruction doesn't appear.

**Comment by JosephineKwa** [comment]
How does this look?
screenshot 33

**Comment by damithc** [comment]
better :-)

**Comment by JosephineKwa** [comment]
Prof @damithc I have just pushed the above changes to the UI and updates to the test cases

**Comment by damithc** [comment]
@kanghj for review and merging please.

File: src/test/resources/pages/newlyJoinedInstructorStudentListPage.html [hunk 1]
```diff
@@ -115,6 +115,10 @@
 
             
 
+            <div>
```
Comment (kanghj) [inline src/test/resources/pages/newlyJoinedInstructorStudentListPage.html]: This is missing a <br> and the "text-muted" class.

**Comment by kanghj** [comment]
Some changed requested. I will merge this after the change is made.

**Comment by JosephineKwa** [comment]
Ok, I have made the necessary changes. Sorry about it!

**Comment by kanghj** [comment]
Looks good. Will merge now.

**Pull request merged**